.github/workflows/pulumi-cli-dev-version.yml

```
@@ -0,0 +1,57 @@
+env:
+  GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
+
+name: Update latest-dev-version
+on:
+  repository_dispatch:
+    types:
+      - pulumi-cli-dev-version
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    needs: update-latest-dev-version
+    steps:
+      - name: checkout docs repo
+        uses: actions/checkout@v2
+      - name: pull-request
+        uses: repo-sync/pull-request@v2
+        with:
+          source_branch: "pulumi/${{ github.run_id }}-${{ github.run_number }}"
+          destination_branch: "master"
+          pr_title: "Regen dev version pulumi@${{ github.event.client_payload.dev_version}}"
+          pr_body: "Automated PR"
+          pr_label: "automation/pulumi-cli-docs,automation/merge"
+          github_token: ${{ secrets.PULUMI_BOT_TOKEN }}
+  build-pulumi-cli-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout docs repo
+        uses: actions/checkout@v2
+      - name: Update latest version
+        run: |
+          echo -n "${{ github.event.client_payload.dev_version }}" > ./static/latest-dev-version
+      - name: git status
+        run: git status && git diff
+      - name: commit changes
+        run: |
+          git config --local user.email "bot@pulumi.com"
+          git config --local user.name "pulumi-bot"
+          git checkout -b pulumi/${{ github.run_id }}-${{ github.run_number }}
+          git add static/
```

Just to scope this to only what we're expecting here:

```suggestion
          git add static/latest-dev-version
```